api/v1alpha1/wildflyserver_types.go

```diff
@@ -38,8 +38,10 @@ type WildFlyServerSpec struct {
 	// SessionAffinity defines if connections from the same client ip are passed to the same WildFlyServer instance/pod each time (false if omitted)
 	SessionAffinity bool `json:"sessionAffinity,omitempty"`
 	// DisableHTTPRoute disables the creation a route to the HTTP port of the application service (false if omitted)
-	DisableHTTPRoute    bool                     `json:"disableHTTPRoute,omitempty"`
-	StandaloneConfigMap *StandaloneConfigMapSpec `json:"standaloneConfigMap,omitempty"`
+	DisableHTTPRoute bool `json:"disableHTTPRoute,omitempty"`
+	// DeactivateTransactionRecovery disables the process of recoverying transactions (false if omitted)
+	DeactivateTransactionRecovery bool `json:"deactivateTransactionRecovery,omitempty"`
```

Comment:
Wrong formatting, generally IDEs do this automatically otherwise you should execute `go fmt ./...` before pushing your changes up.

Reply:
> Wrong formatting, generally IDEs do this automatically otherwise you should execute `go fmt ./...` before pushing your changes up.

Great to know! I will update the PR with `go fmt ./...`. Thanks Yeray :-)

Reply:
Done. Thanks again for the sharing that command :partying_face: